.github/workflows/build_and_deploy.yml

```diff
@@ -0,0 +1,79 @@
+name: Build and deploy to AKS cluster
+
+on:
+  push:
+    branches:
+    - main
+  pull_request:
+    branches:
+    - main
+
+
+concurrency: deploy-${{ github.ref }}
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    outputs:
+      docker-image-tag: ${{ steps.build-image.outputs.tag }}
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+
+    - name: Build and push docker image
+      id: build-image
+      uses: DFE-Digital/github-actions/build-docker-image@master
+      with:
+        github-token: ${{ secrets.GITHUB_TOKEN }}
+        context: .
+
+  deploy_review :
+        name: Deploy to review environment
+        runs-on: ubuntu-latest
+        if: ${{ contains(github.event.pull_request.labels.*.name, 'deploy') }}
+        needs: [build]
+        environment:
+          name: review
+          url: ${{ steps.get-app-url.outputs.url1 }}
+
+        steps:
+          - name: Checkout code
+            uses: actions/checkout@v3
+
+          - uses: ./.github/actions/deploy-environment
+            id: deploy
+            with:
+              environment: review
+              docker-image: ${{ needs.build.outputs.docker-image-tag }}
+              azure-credentials: ${{ secrets.AZURE_CREDENTIALS }}
+              pull-request-number: ${{ github.event.number }}
+
+          - uses: azure/login@v1
+            with:
+              creds: ${{ secrets.AZURE_CREDENTIALS }}
+
+          - name: Seed Review App
+            shell: bash
+            if: github.event.number != ''
+            run: |
+                az aks get-credentials -g s189t01-tsc-ts-rg -n s189t01-tsc-test-aks
+                kubectl exec -n bat-qa deployment/itt-mentor-services-${{ github.event.number }} -- sh -c "export DISABLE_DATABASE_ENVIRONMENT_CHECK=1 && GIAS_CSV_BASE_URL=https://ea-edubase-api-prod.azurewebsites.net/edubase/downloads/public  && cd /app && /usr/local/bin/rake db:seed"
```

Comment:
Is DISABLE_DATABASE_ENVIRONMENT_CHECK needed?

Reply:
This is not required , removed it.